groups/bsl/bslstl/bslstl_optional.h

```diff
@@ -3910,20 +3609,29 @@ class optional<TYPE, false> {
 // }}} END GENERATED CODE
 #endif
 
-    //MANIPULATORS
+    // MANIPULATORS
 #if !BSLS_COMPILERFEATURES_SIMULATE_CPP11_FEATURES
     template <class... ARGS>
     void emplace(BSLS_COMPILERFEATURES_FORWARD_REF(ARGS)...);
-        // Destroy the current 'value_type' object, if any, and create a new
-        // one using the provided arguments.
+        // Assign to this 'optional' object the value of the (template
+        // parameter) 'TYPE' created in place using the specified 'il' and
```

Comment:
There's no `il` argument here.  Please fix here and in other similar places.

Reply:
done